protocol/avdl/chat1/local.avdl

```diff
@@ -105,6 +105,13 @@ protocol local {
     keybase1.TLFIdentifyBehavior identifyBehavior;
   }
 
+  record Inbox {
```

Comment:
Not in the protocol, but it gets used in inboxsource.go. I could just make it a Go type if you think that is better.